tox.ini

```diff
@@ -52,10 +62,21 @@ commands =
 
 [testenv:integration]
 deps =
-    #git+https://github.com/juju/python-libjuju.git
-    git+https://github.com/sed-i/python-libjuju.git@feature/wait_for_status
+    git+https://github.com/juju/python-libjuju.git
     pytest
     git+https://github.com/charmed-kubernetes/pytest-operator.git
 commands =
     pytest -v --tb native --log-cli-level=INFO {posargs} {toxinidir}/tests/integration
 
+[testenv:static]
+deps =
+    -r{toxinidir}/requirements.txt
+    mypy
+    types-PyYAML
+    pytest
+    git+https://github.com/charmed-kubernetes/pytest-operator.git
+    git+https://github.com/juju/python-libjuju.git
```

Comment:
Why the git links? In my testing simply juju and pytest-operator work fine

Reply:
They fixed it! Reverting to pypi